pkg/daemon/update.go

```diff
@@ -1239,7 +1239,9 @@ func (dn *Daemon) isPathInDropins(path string, systemd *ign3types.Systemd) bool
 //
 //nolint:gocyclo
 func (dn *Daemon) deleteStaleData(oldIgnConfig, newIgnConfig ign3types.Config) error {
+
```

Comment:
nit: there seems to be some extra whitespace added here, would you be able to remove?